### ISSUE TITLE
is-locked-open doesn't set correct position attribute

### ISSUE THREAD

**Issue by kilianc** [angular/material#517]
```html
<md-sidenav class="md-whiteframe-z1" is-locked-open="true">
```

doesn't do anything

**Comment by blandine** [comment]
Hi,
It seems that it works with "md-is-locked-open" instead of "is-locked-open".
If you want to test, here is a plunkr : http://plnkr.co/edit/edNexpSixMAtKDyJb0SP?p=preview

**Comment by PaulMougel** [comment]
Didn't look at the actual issue, but I just wanted to point out that eb2f2f8a8c668142742e4b4c1e18cf6d91a533db (available in 0.6.0-rc1 and upwards) introduced the `md-` prefixe that @blandine is talking about. This commit was added *after* this issue was filed, so it probably isn't what fixed the issue, if it is indeed fixed.
